src/coreclr/hosts/corerun/corerun.cpp

```diff
@@ -147,11 +147,11 @@ static string_t build_tpa(const string_t& core_root, const string_t& core_librar
     return tpa_list.str();
 }
 
-static bool try_get_export(pal::mod_t mod, const char* symbol, void** fptr)
+static bool try_get_export(pal::mod_t mod, const char* symbol, void** fptr, bool isOptional = false)
```

Comment:
We're not checking the return value for the case where this is optional. I don't think this is needed if we aren't even checking the result. I'd prefer to just ignore the case at the callsite instead of making an optional param that just confuses the usage. The error message below can stay because it is appropriate and makes sense to report that regardless of intent since this is corerun and for testing.

Reply:
Ok, I was thinking that the error message might confuse people, but I guess it is fine to keep it. The optional argument was added just to suppress it.